disasm.ml

```diff
@@ -14,8 +14,8 @@ let disassemble_annotated (prog : Scope.annotated_program) =
     in
     let str_from_vars vars = String.concat ", " (Scope.VarSet.elements vars) in
     begin match annot with
-    | Some (Scope.Exact e)    -> pr buf "{%s} "      (str_from_vars e)
-    | Some (Scope.At_least e) -> pr buf "{%s, ...} " (str_from_vars e)
+    | Some (Scope.Exact e)    -> pr buf "{%s}"      (str_from_vars e)
+    | Some (Scope.At_least e) -> pr buf "{%s, ...}" (str_from_vars e)
```

Comment:
not sure about this change; wouldn't you get `{x, y}foo:`, with no space in between, in the label case?

We could, of course, also align the printing of the the instructions (by precomputing the scope part, computing the largest scope annotation in the program, and padding with it), but that sounds overkill.